apps/web/src/pages/get-started/components/GetStartedAnimation.tsx

```diff
@@ -0,0 +1,91 @@
+import { FC, useEffect } from 'react';
+import { useRive, useStateMachineInput, decodeFont, FontAsset } from '@rive-app/react-canvas';
+import { useTheme } from '@emotion/react';
+import { OnboardingUseCasesTabsEnum } from '../consts/OnboardingUseCasesTabsEnum';
+import styled from '@emotion/styled';
+
+interface IGetStartedAnimationProps {
+  useCase: OnboardingUseCasesTabsEnum;
+}
+
+// uses `public` as the default base directory
+const ROOT_ANIMATION_PATH = `animations/get-started`;
+const STATE_MACHINE_NAME = 'SM';
+const STATE_MACHINE_INPUT_NAME = 'white theme';
+
+const getAnimationPath = (useCase: OnboardingUseCasesTabsEnum) => `${ROOT_ANIMATION_PATH}/${useCase}.riv`;
+
+const AnimationContainer = styled.div`
+  /* taken from Figma to try to get a good estimate on aspect ratio */
+  aspect-ratio: 540 / 472;
```

Comment:
I would suggest additionally setting the max-width (you can decide about the value, IMO 62.5rem looks fine) because on a bigger screen, it looks huge. And maybe margin auto on a sides to center it.

Reply:
Good suggestions, thanks! In the future, I think we should ask Nikolay to specify during the design phase